lang/java/avro/src/test/java/org/apache/avro/TestSchemaValidation.java

```diff
@@ -121,12 +303,32 @@ public void testInvalidBuild() {
     builder.strategy(null).validateAll();
   }
 
+  Schema s = SchemaBuilder.record("r").fields()
```

Comment:
I cannot see the purpose of this one. Does anyone use `TestSchemaValidation.s`?

Reply:
My mistake, thanks for pointing this out. Updated commit removed the outer s (which was never used in the first place)